modules/abstract-utxo/src/transaction/descriptor/verifyTransaction.ts

```diff
@@ -1,50 +1,54 @@
 import * as utxolib from '@bitgo/utxo-lib';
-import { ITransactionRecipient, VerifyTransactionOptions } from '@bitgo/sdk-core';
+import {
+  ITransactionRecipient,
+  MismatchedRecipient,
+  TxIntentMismatchError,
+  TxIntentMismatchRecipientError,
+  VerifyTransactionOptions,
+} from '@bitgo/sdk-core';
 import { DescriptorMap } from '@bitgo/utxo-core/descriptor';
 
 import { AbstractUtxoCoin, BaseOutput, BaseParsedTransactionOutputs } from '../../abstractUtxoCoin';
 
 import { toBaseParsedTransactionOutputsFromPsbt } from './parse';
 
-export class ValidationError extends Error {
-  constructor(message: string) {
-    super(message);
-  }
-}
-
-export class ErrorMissingOutputs extends ValidationError {
-  constructor(public missingOutputs: BaseOutput<bigint | 'max'>[]) {
-    super(`missing outputs (count=${missingOutputs.length})`);
-  }
-}
-
-export class ErrorImplicitExternalOutputs extends ValidationError {
-  constructor(public implicitExternalOutputs: BaseOutput<bigint | 'max'>[]) {
-    super(`unexpected implicit external outputs (count=${implicitExternalOutputs.length})`);
-  }
-}
-
```

Comment:
You are removing useful structure from the errors here.

Instead of rewriting this file, consider catching the error on in `abstractUtxoCoin.ts` and re-throwing it wrapped in a `TxIntentMismatchError`

If it is important that the outer error is of class `TxIntentMismatchRecipientError`, please wrap the structured error inside. Consider passing it as cause: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/cause